static/sass/application.scss

```diff
@@ -128,3 +126,58 @@
         transform: rotate(180deg)
     }
 }
+
+/*Datepicker overrides*/
+.ui-widget-header {
+    border: 0 !important;
+    background:govuk-colour("white") !important;
+    color: govuk-colour("black") !important;
+    font-weight: bold;
+}
+
+.ui-datepicker {
+    margin-top: 20px;
+}
+
+.ui-datepicker table {
+    background: govuk-colour("white") !important;
+}
+
+.ui-widget-content {
+    border: 1px solid govuk-colour("black") !important;
+    background: govuk-colour("white") !important;
+}
+
+.ui-state-default {
+    border: 0 !important;
+    background: govuk-colour("white") !important;
+    color: govuk-colour("black") !important;
+    font-weight: normal !important;
+    text-align: center !important;
+}
+
+.ui-state-active {
+    background:  $govuk-brand-colour !important;
+    color: govuk-colour("white") !important;
+    border-radius: 15px !important;
+    padding: 2px !important;
+    width: 25px !important;
+    margin: 0 auto !important;
+}
+
+.ui-state-highlight {
+    border: 1px solid #80a9ca !important;
```

Comment:
why the custom colour here?

Reply:
Because it slipped the net.  The little tinker that it is.